txtdirect.go

```diff
@@ -362,6 +365,14 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 		return nil
 	}
 
+	if rec.Type == "dockerv2" {
+		err := redirectDockerv2(w, r, rec)
+		if err != nil {
+			panic(err)
```

Comment:
Please try to avoid panic. It seems that this will crash the program. Using panics and recover in an Exception style pattern is very expensive.

Reply:
It's just for testing, will remove all the unnecessary logs and panics at the end.